Library/Formula/notmuch.rb

```diff
@@ -41,5 +43,11 @@ def install
     system "./configure", *args
 
     system "make", "V=1", "install"
+    if build.with? "python-bindings"
+      Dir.chdir("bindings/python/") do
+        system "python", "setup.py", "install", "--user"
```

Comment:
(or `system python, *Language::Python.setup_install_args(prefix)`, without the quotes around `python`, inside each_python)